<?xml version="1.0" encoding="UTF-8" ?>
<testExecutions version="1">
  <file path="/home/ds/Projects/dfinance/pontem-types/src/index.test.ts">
    <testCase name="Pontem Definitions Pontem Snapshot" duration="3" />
    <testCase name="Pontem Definitions Pontem Types" duration="2">
      <failure
        message="Error: expect(received).toBeFalsy()"
      ><![CDATA[Error: expect(received).toBeFalsy()

Received: "AccountInfoWithDualRefCount"
    at Object.<anonymous> (/home/ds/Projects/dfinance/pontem-types/src/index.test.ts:39:33)
    at Promise.then.completed (/home/ds/Projects/dfinance/pontem-types/node_modules/jest-circus/build/utils.js:390:28)
    at new Promise (<anonymous>)
    at callAsyncCircusFn (/home/ds/Projects/dfinance/pontem-types/node_modules/jest-circus/build/utils.js:315:10)
    at _callCircusTest (/home/ds/Projects/dfinance/pontem-types/node_modules/jest-circus/build/run.js:218:40)
    at processTicksAndRejections (internal/process/task_queues.js:95:5)
    at _runTest (/home/ds/Projects/dfinance/pontem-types/node_modules/jest-circus/build/run.js:155:3)
    at _runTestsForDescribeBlock (/home/ds/Projects/dfinance/pontem-types/node_modules/jest-circus/build/run.js:66:9)
    at _runTestsForDescribeBlock (/home/ds/Projects/dfinance/pontem-types/node_modules/jest-circus/build/run.js:60:9)
    at run (/home/ds/Projects/dfinance/pontem-types/node_modules/jest-circus/build/run.js:25:3)]]></failure>
    </testCase>
  </file>
</testExecutions>
